pkg/build/controller/policy/policy.go

```diff
@@ -139,9 +140,8 @@ func handleComplete(lister buildclient.BuildLister, updater buildclient.BuildUpd
 	if hasRunningBuilds || len(nextBuilds) == 0 {
 		return nil
 	}
-	now := unversioned.Now()
 	for _, build := range nextBuilds {
-		build.Status.StartTimestamp = &now
+		build.Annotations[buildapi.BuildAcceptedAnnotation] = uuid.NewRandom().String()
```

Comment:
Add a TODO for this block - "replace with informer notification requeueing in the future"